src/main/java/toughasnails/temperature/MaterialTemperatureData.java

```diff
@@ -0,0 +1,7 @@
+package toughasnails.temperature;
+
+public class MaterialTemperatureData {
+
+	public float FIRE = 1.0F;
+	
```

Comment:
The only material that I was planning to do this for was Materials.FIRE, as it doesn't really make sense for the other materials except Materials.LAVA, and that can be achieved by specifying temperatures for Blocks.LAVA and Blocks.FLOWING_LAVA (with the advantage that source blocks and flowing blocks can be treated differently if desired).

I couldn't find a way of mapping from a string to a material name cleanly (Material.toString() isn't intelligible and I believe would change if properties of a material were to change), otherwise I would have allowed it to be specified for all materials anyway, even if it might not be wanted.

I will change it to lowercase.